client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -134,6 +135,10 @@ export class DomainDetailsForm extends PureComponent {
 		return this.state.currentStep !== last( this.state.steps );
 	}
 
+	isInternationalAddress() {
+		return ! includes( CHECKOUT_US_ADDRESS_FORMAT_COUNTRIES, formState.getFieldValue( this.state.form, 'countryCode' ) );
```

Comment:
If we pass the countryCode in we can make this function a easier to re-use.

Reply:
Or we could test this condition via a selector in `connect()` and pass down as a prop. 
This way we'd only need to run it once as well. (?)